scio-bigquery/src/main/scala/com/spotify/scio/bigquery/types/SchemaProvider.scala

```diff
@@ -49,24 +51,27 @@ private[types] object SchemaProvider {
   }
 
   // scalastyle:off cyclomatic.complexity
-  private def rawType(tpe: Type): (String, Iterable[TableFieldSchema]) = tpe match {
-    case t if t =:= typeOf[Boolean] => ("BOOLEAN", Iterable.empty)
-    case t if t =:= typeOf[Int] => ("INTEGER", Iterable.empty)
-    case t if t =:= typeOf[Long] => ("INTEGER", Iterable.empty)
-    case t if t =:= typeOf[Float] => ("FLOAT", Iterable.empty)
-    case t if t =:= typeOf[Double]  => ("FLOAT", Iterable.empty)
-    case t if t =:= typeOf[String] => ("STRING", Iterable.empty)
-
-    case t if t =:= typeOf[ByteString] => ("BYTES", Iterable.empty)
-    case t if t =:= typeOf[Array[Byte]] => ("BYTES", Iterable.empty)
-
-    case t if t =:= typeOf[Instant] => ("TIMESTAMP", Iterable.empty)
-    case t if t =:= typeOf[LocalDate] => ("DATE", Iterable.empty)
-    case t if t =:= typeOf[LocalTime] => ("TIME", Iterable.empty)
-    case t if t =:= typeOf[LocalDateTime] => ("DATETIME", Iterable.empty)
-
-    case t if isCaseClass(t) => ("RECORD", toFields(t))
-    case _ => throw new RuntimeException(s"Unsupported type: $tpe")
+  private def rawType(tpe: Type): (String, Iterable[TableFieldSchema]) = {
+    val provider: ValidationProvider = ValidationProviderFinder.getProvider
```

Comment:
Is it possible to minimize the re-indentation change here by moving `val provider` outside of the method?

Reply:
yeah it is the scope is slightly bigger but not a big deal